### PR TITLE
Rename cli capsule tests as per issue #3083

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: tests/foreman/cli/test_capsule.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def setUp(self):
 
     @run_only_on('sat')
     @tier1
-    def test_verify_redmine_3875(self):
+    def test_positive_create_with_random_url(self):
```
Comment (omaciel) [inline tests/foreman/cli/test_capsule.py]: I'd remove `random` as we have many tests using random values but we don't explicitly call it out.

Comment (sthirugn) [inline tests/foreman/cli/test_capsule.py]: :+1:

File: tests/foreman/cli/test_capsule.py [hunk 2]
```diff
@@ -210,7 +210,7 @@ def test_positive_consume_content_through_capsule(self):
         """
 
     @stubbed()
-    def test_positive_unsusbscribe_content_through_capsule(self):
+    def test_positive_unsusbscribe(self):
```
Comment (omaciel) [inline tests/foreman/cli/test_capsule.py]: Typo from before: s/unsusbscribe/unsubscribe

**Comment by sthirugn** [comment]
Comments updated and rebased

**Comment by omaciel** [comment]
ACK

**Pull request merged**